dashboard/app/models/levels/aichat.rb

```diff
@@ -24,14 +24,11 @@
 #  index_levels_on_name       (name)
 #
 
-# TODO: Update this level to account for the params we actually need.
 class Aichat < Level
   serialized_attrs %w(
-    project_template_level_name
```

Comment:
These don't seem like attributes we'll need for these levels, but I could be wrong? Easy enough to re-add if we do.

Reply:
Yeah, Brendan helped confirm when I originally asked/double-checked that any of the serialized_attrs can changed later: https://github.com/code-dot-org/code-dot-org/pull/52538/files#r1260226028